awx/ui_next/src/screens/Template/JobTemplateSchedule/JobTemplateSchedule.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
would it be possible for us to make this generic like we have for resource access list (where you pop in the list into the resource, and that gives you the ability to add/edit roles).  I understand that is not quite the same, because all that stuff is in a modal, and not part of the router.  But if we could basically generate all these "sub routes" by including the ScheduleList, it would make sharing this across routes quite a bit easier, I think. I think we'll need this in 4 places, at the root, on JT, WFJT and Project.